src/routes/projects/+page.svx

```diff
@@ -73,6 +73,10 @@ An Open Brewery DB GraphQL server implementation in .NET Core and maintained by
 
 A Python interface for the API built and maintained by [@jrbourbeau](https://github.com/jrbourbeau/)
 
+#### [BrewFind Django App](https://github.com/Rmariner25/moengage_brewery)
+
+A Django-based web application to search, add review, and view reviews for breweries. Implemented search autocomplete feature among other features such as caching api results, pagination, embeded Google Maps for breweries. [@jrbourbeau](https://github.com/Rmariner25)
```

Comment:
`[@jrbourbeau](https://github.com/Rmariner25)` should probably be `[@Rmariner25](https://github.com/Rmariner25)` :)

Reply:
Sorry for the oversight. Thanks for the merge.